src/utils/ucc_proc_info.c

```diff
@@ -3,6 +3,10 @@
 #include "ucc_log.h"
 #include <limits.h>
 #include <stdint.h>
+#include "config.h"
```

Comment:
i remember we decided to put all the UCS stuff to utils. So, lets create ucc_get_system_id function, and implement it there via ucs_get_system_id.